### DISCUSSION TITLE
插件安装后不显示  主笔记按钮  无法设置主笔记

### DISCUSSION THREAD

**Discussion by zhongyuchen808** [windingwind/zotero-better-notes#1021] [Q&A] [answered]
插件安装后不显示  主笔记按钮  无法设置主笔记插件安装后不显示  主笔记按钮  无法设置主笔记

**Comment by windingwind** [comment, marked as answer]
最新版本不再有主笔记概念，任何笔记都可以是过去的所谓“主笔记”，也无需设置。

**Reply by zhongyuchen808** [reply]
感谢~~~~

**Reply by cheny1ing** [reply]
最新版本无主笔记的概念很好，但是使用“链接精灵”的效率没有一键关联到主笔记的效率高，可否也保留主笔记的按钮或者快速添加的按钮？

**Reply by cheny1ing** [reply]
最新版本的思维导图如何调出？

**Reply by windingwind** [reply]
请勿在不相关问题下提问其他内容@cheny1ing

**Reply by xusatellite** [reply]
伟大的better note开发者，您好。前期版本的主笔记功能，很大的优点在于(1) 可以把零散的笔记很方便的积累起来，(2) 阅读过程中的笔记可以通过很少的点击快速记录进主笔记中，这很方便后续查看积累的内容。现在如何实现这两点呢？谢谢您。

**Reply by windingwind** [reply]
后者可以通过新版本的链接精灵实现

**Reply by xusatellite** [reply]
> 后者可以通过新版本的链接精灵实现

链接精灵会打开新窗口，需要检索主笔记(检索存在多条的话需要翻找，这点很不方便)，然后选择插入位置，再插入。多次步骤和切换界面没有6.0版本的简洁方便。建议仅供参考，感谢您，无私的开发者。
